index.js

```diff
@@ -50,6 +50,13 @@ var proxies = [
 ];
 
 /**
+ * Regex to match any comma surrounded by optional whitespace
+ *
+ * @type {RegExp}
+ */
+var ipListSpaceFilter = /\s*,\s*/;
```

Comment:
@jcrugzz correct. It should have been addressed [here](https://github.com/primus/forwarded-for/commit/2f9bd0ed2b1122b07a821abd49f49d238e682ae1).